### PR TITLE
Fixes tabs to reflect that this also works on Mac

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8695563](https://github.com/OfficeDev/office-js-docs-pr/commits/8695563a7f0fafa192ab6da9699bc60b64daea23)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.vscode/settings.json](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixtutmac/.vscode/settings.json) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixtutmac/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908072212087968-1229%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=1S47KQ24ryJBmvIKg2%2BPFz0mz8XGoztTQwnn8JJSfoc%3D&st=2019-08-07T22%3A08%3A12Z&se=2019-09-07T22%3A13%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tutorials/excel-tutorial-create-custom-functions.md [hunk 1]
```diff
@@ -142,9 +142,9 @@ Integrating data from the Web is a great way to extend Excel through custom func
     npm run build
     ```
 
-4. Complete the following steps (for either Excel on the web or Windows) to re-register the add-in in Excel. You must complete these steps before the new function will be available.
+4. Complete the following steps (for either Excel on the web or Windows or Mac) to re-register the add-in in Excel. You must complete these steps before the new function will be available.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-tutorial-create-custom-functions.md]:
```suggestion
4. Complete the following steps (for Excel on the web, Windows, or Mac) to re-register the add-in in Excel. You must complete these steps before the new function will be available.
```

File: docs/tutorials/excel-tutorial-create-custom-functions.md [hunk 2]
```diff
@@ -214,7 +214,7 @@ function clock(invocation) {
 
 3. Complete the following steps (for either Excel on the web or Windows) to re-register the add-in in Excel. You must complete these steps before the new function will be available.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-tutorial-create-custom-functions.md]: You likely need to make a similar change here.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[39289d1](https://github.com/OfficeDev/office-js-docs-pr/commits/39289d10f9d198bff1319d56ae08ec8caade6b63)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.vscode/settings.json](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixtutmac/.vscode/settings.json) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixtutmac/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908072344150869-1229%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=hz3r%2FCijWS9LJG%2FGwteNxIAWb2mL2z76YFaSpokBXQw%3D&st=2019-08-07T23%3A40%3A18Z&se=2019-09-07T23%3A45%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tutorials/excel-tutorial-create-custom-functions.md (hunk 1 above)
Comment (mscharlock) [inline docs/tutorials/excel-tutorial-create-custom-functions.md]:
```suggestion
4. Complete the following steps (for either Excel on the web, Windows, or Mac) to re-register the add-in in Excel. You must complete these steps before the new function will be available.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[39289d1](https://github.com/OfficeDev/office-js-docs-pr/commits/39289d10f9d198bff1319d56ae08ec8caade6b63)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.vscode/settings.json](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixtutmac/.vscode/settings.json) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixtutmac/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908072344264776-1229%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=87YeTIfxEIXqJhLVs8AzGHL5foNTxT6y7i1MGm%2FMphk%3D&st=2019-08-07T23%3A41%3A18Z&se=2019-09-07T23%3A46%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tutorials/excel-tutorial-create-custom-functions.md (hunk 2 above)
Comment (mscharlock) [inline docs/tutorials/excel-tutorial-create-custom-functions.md]:
```suggestion
3. Complete the following steps (for either Excel on the web, Windows, or Mac) to re-register the add-in in Excel. You must complete these steps before the new function will be available. 
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[43f316f](https://github.com/OfficeDev/office-js-docs-pr/commits/43f316fe25d7ab5ceee24253c663a5fbbd0e8d1c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.vscode/settings.json](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixtutmac/.vscode/settings.json) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixtutmac/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908072346437004-1229%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=kCprsLK9p5iTB3KxU5f9hrP4s1jBDx5g0I3ezb8BQw8%3D&st=2019-08-07T23%3A42%3A42Z&se=2019-09-07T23%3A47%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tutorials/excel-tutorial-create-custom-functions.md [hunk 3]
```diff
@@ -212,9 +212,9 @@ function clock(invocation) {
     npm run build
     ```
 
-3. Complete the following steps (for either Excel on the web or Windows) to re-register the add-in in Excel. You must complete these steps before the new function will be available. 
+3. Complete the following steps (for either Excel on the web, Windows, or Mac) to re-register the add-in in Excel. You must complete these steps before the new function will be available.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-tutorial-create-custom-functions.md]:
```suggestion
3. Complete the following steps (for Excel on the web, Windows, or Mac) to re-register the add-in in Excel. You must complete these steps before the new function will be available. 
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8dcf1e3](https://github.com/OfficeDev/office-js-docs-pr/commits/8dcf1e34c06ea30aa9f02f3c42d427dd6507f0f3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.vscode/settings.json](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixtutmac/.vscode/settings.json) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixtutmac/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908081830142846-1229%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2kAnkWcMCLtb3%2BtzR3D1ot4P9r%2FPM8D%2FVNPXAVct6gw%3D&st=2019-08-08T18%3A26%3A18Z&se=2019-09-08T18%3A31%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**